client/src/pages/private/ParticipantDetailsView.js

```diff
@@ -280,6 +307,38 @@ export default () => {
               </Grid>
             </Box>
 
+            {/* Participant RoS Info */}
+            <CheckPermissions
+              permittedRoles={['health_authority', 'employer', 'ministry_of_health']}
+            >
+              {participant.ros && (
+                <>
+                  <Box pt={4} pb={2} pl={4} pr={4}>
+                    <Typography variant='h2'>Return of Service</Typography>
+                  </Box>
+
+                  <Box py={2} px={4}>
+                    <Grid className={classes.rootContainer} container spacing={2}>
+                      {Object.keys(rOSKeyMap).map((key) => (
+                        <Grid key={key} item xs={12} sm={6} xl={3}>
+                          <Grid item xs={6}>
+                            <Typography variant='body1'>
+                              <b>{rOSKeyMap[key]}</b>
+                            </Typography>
+                          </Grid>
+                          <Grid item xs={6}>
+                            <Typography test-id={'participantDetailsRosView' + key} variant='body1'>
```

Comment:
Do we need `test-id` here?

Reply:
This is for future testability, once our data setup issue will fix, we will implement a test for this section.